server/src/library/externalLibraryUtils.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as PQLS from "@microsoft/powerquery-language-services";
+
+export type ExternalSymbolLibrary = ReadonlyArray<PQLS.LibrarySymbol.LibrarySymbol>;
+export type IncomingExternalSymbolLibrary = ExternalSymbolLibrary | undefined | null;
+
+export function getSymbols(): ExternalSymbolLibrary[] {
+    return Array.from(externalLibraryByName.values());
+}
+
+export function setRange(symbols: [string, IncomingExternalSymbolLibrary][]): void {
```

Comment:
ReadonlyMap

Reply:
It doesn't look like the JSON-RPC libraries support serializing typescript Maps. I'm keeping the tuple, but changed it to use a `ReadonlyArray`.